test/IntegrationTestCommon/IntegrationTestCommon.csproj

```diff
@@ -13,5 +13,6 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Identity\Identity.csproj" />
     <ProjectReference Include="..\Common\Common.csproj" />
+    <ProjectReference Include="..\..\bitwarden_license\src\Scim\Scim.csproj" />
```

Comment:
Please do not have the common project depend on scim. Move the factory to the scim project.

Reply:
Makes sense! I've updated the branch with your suggestion.